node-watcher/src/tasks/createTipStatus.ts

```diff
@@ -77,7 +77,8 @@ const createTipStatus: Task<NomidotTipStatusUpdate[]> = {
         let closes: number | undefined;
 
         if (method === tipStatus.CLOSING) {
-          const tipInfoRaw: Option<OpenTip>  = await api.query.treasury.tips(
+          const tipInfoRaw: Option<OpenTip>  = await api.query.treasury.tips.at(
```

Comment:
but this tip is at this blockHash?

Reply:
this is create tip status

Reply:
if you don't specify the block, it takes the head. So when you are in sync with the chain, that's fine, when you're not, it throws because nothing was found. Try it out. Any query must have `.at`, check it out, they all have it.

Reply:
just to add to the above explanation:
Say the harvester is at block 1, the chain (top is at block 99).
- the block 1 has this tipStatus change event.
- you check the state of the blockchain for this tip hash
- but the block 99 is far in the future, at 99 the tip was closed and got removed from the state
- it you don't `query.at(1)`, the `query` will be at block 99. And the state being empty, we go into the following throw `no hash found...`

I ran into it myself, when testing the api bump. I had a long chain with tips, and closed tips. I reset the chain-db and launch the harvester. you then run into the issue.